lhotse/dataset/speech_recognition.py

```diff
@@ -42,9 +42,8 @@ def __getitem__(self, idx: int) -> Dict[str, torch.Tensor]:
 
         features = torch.from_numpy(cut.load_features(root_dir=self.root_dir))
 
-        # We assume there is only one supervision for each cut in ASR tasks (for now)
-        if len(cut.supervisions) > 1:
-            logging.warning("More than one supervision in ASR task! Selected the first one and ignored others.")
+        # There must be only one supervision because we have had trim_to_supervisions() processed
+        assert len(cut.supervisions) == 1
```

Comment:
```suggestion
        assert len(cut.supervisions) == 1, "SpeechRecognitionDataset does not support overlapping supervisions yet."
```

Reply:
I think this message will never be printed because `trim_to_supervisions()` makes sure that there would be only one supervision. So can we set this message as: "Unexpected multi-supervisions cut. Maybe cuts.trim_to_supervisions() was not work correctly."